tethys_gizmos/static/tethys_gizmos/js/jobs_table.js

```diff
@@ -3,135 +3,73 @@
  * Update job status with a timeout while job is still running.
  *
  *****************************************************************************/
-
-function bind_run_button(btn){
-    var job_id = $(btn).data('job-id');
-    $(btn).on('click', function () {
-        var execute_url = '/developer/gizmos/ajax/' + job_id + '/execute';
-        $.ajax({
-            url: execute_url
-        }).done(function (json) {
-            status_html =
-            '<div class="progress" style="margin-bottom: 0;">' +
-                '<div class="progress-bar bg-warning progress-bar-striped active" role="progressbar" title="Submitted" aria-valuenow="100" aria-valuemin="0" aria-valuemax="100" style="width: 100%">' +
-                    '<span class="sr-only">100% Complete</span>' +
-                '</div>' +
-            '</div>'
-            $(btn).parent().html(status_html);
-            update_row($('#jobs-table-row-' + job_id));
-        });
-    });
+function add_message(message, message_type='danger'){
+  if(message){
+    if(message_type == 'danger'){
+      message = '<strong>Error!</strong> ' + message;
+    }
+    var alert_html = '<div class="alert alert-' + message_type + ' alert-dismissible" role="alert">' +
+                        '<button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>' +
+                         message +
```

Comment:
Update dismiss button to bootstrap 5:

```suggestion
                         message +
                        '<button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>' +
```